clicommand/artifact_search.go

```diff
@@ -145,10 +145,8 @@ var ArtifactSearchCommand = cli.Command{
 			return err
 		}
 
-		artifactCount := len(artifacts)
-
-		if artifactCount == 0 {
-			l.Debug("No matches found for %s", cfg.Query)
+		if len(artifacts) == 0 {
+			return cli.Exit(fmt.Sprintf("No matches found for %s", cfg.Query), 1)
```

Comment:
Maybe `%q` instead of `%s` to get the string nicely quoted? No big deal either way, though.